src/fides/config/config_proxy.py

```diff
@@ -163,6 +170,7 @@ class ConfigProxy:
     """
 
     def __init__(self, db: Session) -> None:
+        self.admin_ui = AdminUISettingsProxy(db)
```

Comment:
if we're putting these in the config proxy, i'm assuming we'd also want to be able to _read_ the values of these settings through the `GET /config` endpoint? if so, they'll need to be added to the `CONFIG_KEY_ALLOWLIST` in `src/fides/config/utils.py`, and there's also some tests that'll need to be updated.

Reply:
nice, thanks for adding 👍